test/test_generate.py

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/python3
+
+import os
+import sys
+from subprocess import check_output
+
+# Validates that certain files are not modified after running "go generate"
+
+# List of directories that contain generate.go files
+list_of_directories=["../idb/postgres/internal/schema"]
+
+for dir in list_of_directories:
+    os.system("/usr/local/go/bin/go generate {}".format(dir))
```

Comment:
Does this work without specifying the full path to `go`? It needs to be on the path for everything else so I think it's a fair assumption as long as it works.

Reply:
It does not work without it unfortunately